### DISCUSSION TITLE
Use of GDS Transport font on products not on service.gov.uk

### DISCUSSION THREAD

**Discussion by kelliedesigner** [ministryofjustice/moj-frontend#249] [RELEASED: Components, pages and patterns]
#Related pages

- [Typography - GOV.UK Design System](https://design-system.service.gov.uk/styles/typography/)
- [Making your service look like GOV.UK](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk)
- [GOV.UK Proposition](https://www.gov.uk/government/publications/govuk-proposition/govuk-proposition)

# Background

Guidance on the GOV.UK Design System states:

> If your service is on the service.gov.uk subdomain you must use the GDS Transport font.
>
> You should use an alternative typeface like Helvetica or Arial for services that are publicly available on different domains.

- GDS policy states you should no use GDS Transport if your service is not on service.gov.uk
- There is no licensing restrictions on the use of GDS Transport.

## Fonts used in other departments

- HMRC - Arial
- Home Office - Roboto
- HMLR - Arial / Helvetica
- DfE - GDS Trasnport
- ESFA - GDS Transport

# MOJ guidance

## HMPPS Probation Designers community discussion 1 September 2021

- There is a feeling in MOJ and across gov that the GDS policy on this is out of date, and should be relaxed so that GDS Transport can be more widely used.
- There is also no enforcement if GDS Transport is used on services not on service.gov.uk

By not using GDS Transport, we miss out on:
- accessibility and readability optimisation
- familiarity of GOV.UK
- seamless use of GOV.UK components and patterns (alternative fonts can break them and cause issues)

### Actions

We will:
- continue to use GDS Transport across probation digital products
- provide MOJ wide guidance / stance on the font in the Pattern Library
- feedback our approach to  GDS, to start the conversation on relaxing the policy

**Comment by JSJohal** [comment]
> You should use an alternative typeface like Helvetica or Arial for services that are publicly available on different domains.

One could argue that our services aren't publicly available as they aren't available to the general public...

Also, something I'd thought about before - if it was ever enforced upon us that we need to change the font across our services as we're breaching the GDS transport license policy, i'd argue to push for a technical change to our URL structure rather than change the font across all the services which would impact our users negatively.
By this I mean, I'd argue for changing our URLs from service.justice.gov.uk to justice.service.gov.uk and handling the change through URL redirects.

**Comment by AndyRead1** [comment]
UK Policing particularly forensic multimedia production units would like to introduce the use of the GDS Transport typeface for producing video compilations and multimedia that is presented in court as evidence. This is to make these presentations more accessible and easier to understand when highlighting aspects on video evidence including CCTV where we may show points of interest or track vehicles or persons of interest. 
So would UK Policing, producing this evidence for the MOJ and CPS be classed as a subdomain or would we have to purchase the font directly from A2 Type, for a national licence to use “New Transport" instead?

**Comment by robertjmccarthy** [comment]
Closing this discussion as an issue has been created #702
